apps/frontend/src/modules/surveys/components/studio/create/editor-panel/questions/attributes/QuestionTypes.tsx

```diff
@@ -26,6 +27,18 @@ export const QuestionOptions = () => {
         );
         setWelcomeMessageExists(!!welcomeMessage);
         setThankYouMessageExists(!!thankYouMessage);
+
+        const handleClickOutside = (event: MouseEvent) => {
+            if (questionTypesRef.current && !questionTypesRef.current.contains(event.target as Node)) {
+                setShowQuestionTypes(false);
+            }
+        };
+
+        document.addEventListener("mousedown", handleClickOutside);
```

Comment:
@afolasope You should create a separate `useEffect` hook for this

Reply:
Okay, I have